analyzer/codechecker_analyzer/analyzers/clangsa/analyzer.py

```diff
@@ -319,6 +323,8 @@ def construct_config_handler(cls, args, context):
         check_env = get_check_env(context.path_env_extra,
                                   context.ld_lib_path_extra)
 
+        handler.enable_z3 = 'enable_z3' in args
```

Comment:
Shouldn't we check for the option like this?
```suggestion
        handler.enable_z3 = 'enable_z3' in args and args.enable_z3
```
If it is available it will be enabled even if the user did not enable it.